### ISSUE TITLE
Misbehaviors in 3.9.0 :7ad9df044: (8175)

### ISSUE THREAD

**Issue by macsrwe** [ViennaRSS/vienna-rss#1722]
Just installed this release onto macOS Monterey 12.7.2. 

Upon launch, main window immediately snapped to entire width of screen, and left column snapped to minimum width; neither would respond to any corrective drag attempts, though cursor changed appropriately.

Relaunching seemed to clear this problem up... but shortly after, window snapped into the same state. Furthermore, typing into input fields in posted articles exhibited a further failure -- after typing a single character, input field grayed out and focus was shifted to the search bar. Typing a return character into an input field grayed out the field and opened the article in an external browser.

**Comment by barijaona** [comment]
To help us sort out the issue, can you quit Vienna, put aside its Preferences file (```~/Library/Preferences/uk.co.opencommunity.vienna2.plist```) and relaunch the app afterwards ?

**Comment by macsrwe** [comment]
OK... I did this, relaunched, it came up looking good in horizontal view with the bottom (article contents) pane closed. I lifted up the pane divider until it was half and half (which is how I usually use it). Then I selected a newsgroup, and an article line in that newsgroup. Bam, the window immediately readjusted itself to full width and minimum left-hand column, immediately as it displayed the article in the bottom pane.

**Comment by barijaona** [comment]
Do you also see a problem similar to #1595 and #1622 : the width of the pane displaying the folder  list changes after Vienna is quit and relaunched ?

**Comment by macsrwe** [comment]
I can't tell if it happens with "every" feed, but it's happened with more than one.

I launched it fresh and bopped around a bit... it took me under 30 seconds to make it happen. That particular feed was 
https://feeds.feedburner.com/LegalInsurrection
and the article was third from last at this moment: "The Los Angles Times Announces Significant Employee Layoffs."

As to @barijaona 's question, those bugs seem to involve the LH column size changing at launch time. In my case, the LH column is something reasonable at launch time, but slams itself to the minimum size upon choosing some article and then cannot be resized in any fashion short of quitting the app, nor can the window width be reduced to less than the new width, which is slightly more than full screen width (it can be expanded though).

It may be significant that I read most of my feeds in "Use Web Page for Articles" mode, and all the misbehavior has occurred so far with one of those feeds. It's exactly as if the app makes a computation when rendering the webpage image that it needs more than the available pane space to display it, and slams the window wide open and the LH column to minimum in an attempt to claim that much space. Just for kicks, after Vienna slammed the window wide open, I tried choosing a different feed that was not in web page mode, to see if the resizing controls would work again, but they didn't. Once they have been set wide, they're stuck until I quit the app.

**Comment by barijaona** [comment]
> Using https://waronguns.com/feed/, open any posting and type stuff in the comment box to see the misbehavior. 

This one is due to the fact that what the user types is intercepted by Vienna before being sent to the web view.
If I type "m" in the comment box, the article gets flagged / unflagged 😆

**Comment by barijaona** [comment]
@macsrwe Have you installed Vienna 3.9.1 ? Any comment on its behavior regarding these issues ?

**Comment by macsrwe** [comment]
Ha! Yes, I got suckered in by the "update available" announcement and absent-mindedly updated it. Started it up and immediately my left hand column shrank to its minimum width and could not be widened. Immediately I recalled that I was not updating Vienna (and why), and reinstalled the safe version I keep in Downloads. I didn't even bother to test the other objectionable behavior, i.e., the lost focus and the interpretation of field input as Vienna shortcut commands.
